contracts/queue/src/lib.rs

```diff
@@ -0,0 +1,38 @@
+pub mod contract;
+
+/** Below we expose wasm exports **/
+#[cfg(target_arch = "wasm32")]
+pub use cosmwasm_std::{allocate, deallocate};
+
+#[cfg(target_arch = "wasm32")]
+pub use wasm::{handle, init};
```

Comment:
I was wondering about this code elsewhere already: if `query` was forgotten here and is not missing, do we need this statement at all?

Reply:
I guess not